src/main/java/org/betonquest/betonquest/item/typehandler/UnbreakableHandler.java

```diff
@@ -1,15 +1,64 @@
 package org.betonquest.betonquest.item.typehandler;
 
+import org.betonquest.betonquest.exceptions.InstructionParseException;
 import org.betonquest.betonquest.item.QuestItem.Existence;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.Set;
 
 @SuppressWarnings("PMD.CommentRequired")
-public class UnbreakableHandler {
+public class UnbreakableHandler implements ItemMetaHandler<ItemMeta> {
+
+    /**
+     * The unbreakable string.
+     */
+    private static final String UNBREAKABLE = "unbreakable";
 
     private Existence unbreakable = Existence.WHATEVER;
 
     public UnbreakableHandler() {
     }
 
+    @Override
+    public Class<ItemMeta> metaClass() {
+        return ItemMeta.class;
+    }
+
+    @Override
+    public Set<String> keys() {
+        return Set.of(UNBREAKABLE);
+    }
+
+    @Override
+    public String serializeToString(final ItemMeta meta) {
+        if (meta.isUnbreakable()) {
+            return " unbreakable";
```

Comment:
I think it would be a good idea to handle the leading space outside, so no one need to think about it.